src/redux/projects/thunks.js

```diff
@@ -1,10 +1,12 @@
 import * as actions from './actions';
 
+const URL = `${process.env.REACT_APP_API_URL}/projects`;
+
 export const getProjects = () => {
   return async (dispatch) => {
     dispatch(actions.getProjectsLoading());
     try {
-      const response = await fetch(`${process.env.REACT_APP_API_URL}/projects`);
+      const response = await fetch(`${URL}`);
```

Comment:
If you like this way, you could do it like this. Only in this function, because you don't need to concat anything
```suggestion
      const response = await fetch(URL);
```